### ISSUE TITLE
Importing txt notes and exporting in-app notes as txt files to folder

### ISSUE THREAD

**Issue by krwow** [spacecowboy/NotePad#548]
**Is your feature request related to a problem? Please describe.**
I miss importing txt notes and exporting in-app notes as txt files. There's no quick way to import notes from outside app, so I cannot switch from the other app with a lot of my notes or import these from desktop. Or in the other way - there's no easy solution to export notes and edit them later on desktop.

**Describe the solution you'd like**
I'd be glad if there's another backup option inside app - to export all notes to selected folder on mobile device. And then the same thing in the another way - to import txt files from selected folder.

**Describe alternatives you've considered**
The most obvious solution now is to send e.g. by e-mail every note, one by one, and then download it on PC. When it comes to import, titles and notes content can be copied manually, which will result in creating new notes inside app, but that will be really much work if there's tens or hundreds of notes. Another approach to use notes from app is to create some script in programming language to get data from json file, but that requires extra work and cannot help with missing txt imort anyway.

**Comment by CampelloManuel** [comment]
Go to Settings -> Syncing -> enable synchronization and SD card sync.
The app will export notes as .org files.
You can open them with any text editor.
If you edit the .org files, you should see the notes change in the app.

**Comment by krwow** [comment]
Thanks for a hint, but I edited note, a file with list of tasks to be exact, and it didn't work - I don't see any change inside app. That might be the issue with the place where these notes are being saved, so "Android/data", with permission to edit files there... I think that adding an option to choose where these notes should be saved/synchronised would greatly help with this issue and make these files visible in every file manager. I had trouble to find the app which shows these files, files in "data" folder are not visible in most of apps.

And second thing is the format of export. I wouldn't say that txt or, it would be okay as well, md extension is not popular. I believe it's the first idea when it comes to notes... That's why next to some "internal" backup format there should be most obvious one, the one which can be edited quickly. I wish there would be an option to export of all notes to separate txt files in one location or even to different subfolders based on tasks/lists belongingness, not just one file for every tasks/lists section.

And I used to take notes with your app a years ago, but after its revival it's like nothing has changed at all. It's like any new or missing features are not on their way to app, what makes me upset. App looks like it looked back then.
